core/templates/dev/head/pages/library/SearchResultsDirective.js

```diff
@@ -26,6 +26,7 @@ oppia.directive('searchResults', [
         function($scope, $rootScope, $timeout, $window, siteAnalyticsService) {
           $rootScope.loadingMessage = 'Loading';
           $scope.someResultsExist = true;
+          $scope.hasFullyRegistered = Boolean(GLOBALS.username);
```

Comment:
Actually, one sec. You mentioned in #3732 that there's GLOBALS.userIsLoggedIn. Could we reuse that instead?

If not, this variable should actually be named $scope.userIsLoggedIn. "hasFullyRegistered" is a slightly different concept.